overlord/snapstate/snapstate.go

```diff
@@ -225,14 +230,18 @@ func doInstall(st *state.State, snapst *SnapState, snapsup *SnapSetup, flags int
 
 	installSet := state.NewTaskSet(tasks...)
 
-	configSet := Configure(st, snapsup.Name(), defaults)
+	var confFlags int
+	if snapsup.Name() == "core" {
```

Comment:
this bit probably merits a comment? checking on name feels always a bit strange, we don't have the type until we actually run the change though